### PR TITLE
[Community] Member Profile: Krishna Gupta

### LINKED ISSUE #3445: [Community] Member Profile: Krishna Gupta
**Current Behavior**

@Krishna2323  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Krishna2323 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @Krishna2323
- Twitter: Web_DevLoving
- LinkedIn: krishnagupta23
- Link to profile:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by tinniaru3005**
Signed-off-by: Arunima Chaudhuri <arunimachaudhuri2020@gmail.com>

**Description**
recognize @Krishna2323 as a contributor and community member by creating a profile on https://layer5.io/community/members.

This PR fixes #3445 
Closes #3445 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by tinniaru3005** [comment]
Hey @Chadha93, let me know if any modifications are required. Will update the remaining details (bio and profile picture) when @Krishna2323 provides it. @Chadha93 can you please guide me on the remaining details (badges to publishes) what to fill up.

**Comment by l5io** [comment]
🚀 Preview for commit e4297789fb4f69dc037010661ff40681f2cc323a at: https://637807a188b1913895425b93--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bfbba3447dd21d4845ca21b7814744fa33d0bf5f at: https://63793c7d6be9ca28a9c825c7--layer5.netlify.app

**Comment by leecalcote** [comment]
@Krishna2323 how's it look?

Thank you, @tinniaru3005!

**Comment by Krishna2323** [comment]
@leecalcote, I can't see my name here😭......looks like I have to provide some details here.....A fault from my side.
Thanks @tinniaru3005🙌

**Comment by tinniaru3005** [comment]
> @leecalcote, I can't see my name here😭......looks like I have to provide some details here.....A fault from my side. Thanks @tinniaru3005🙌

Also, let me know if the other details are fine or not. I will update the PR accordingly. :)

**Review by Krishna2323 — COMMENTED** [review]
@tinniaru3005 pls change the status to active. I will share the bio and display pic today.

**Comment by tinniaru3005** [comment]
> @tinniaru3005 pls change the status to active. I will share the bio and display pic today.

Ya, I did that. :)

**Comment by l5io** [comment]
🚀 Preview for commit 382fcf315cadddfd1925b6aa194814faff6a5193 at: https://6379e84f82dd300f48a34f38--layer5.netlify.app

File: src/collections/members/krishna-gupta/index.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Krishna Gupta
+position: Contributor
+image_path: ./display-picture.png
```
Comment (leecalcote) [inline src/collections/members/krishna-gupta/index.mdx:4]: The failing build is likely due to this missing image.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/collections/members/krishna-gupta/index.mdx (hunk 1 above)
Comment (tinniaru3005) [reply src/collections/members/krishna-gupta/index.mdx:4]: Ya I will add it when @Krishna2323 gives the image and bio

**Comment by eeshaanSA** [comment]
@Krishna2323 is this ready?

**Comment by eeshaanSA** [comment]
@tinniaru3005 the build is failing. Are all the fields entered correctly? Image, links etc?

**Comment by tinniaru3005** [comment]
> @tinniaru3005 the build is failing. Are all the fields entered correctly? Image, links etc?

No they are not. Will update it once @Krishna2323 updates his details.

**Comment by Krishna2323** [comment]
@tinniaru3005 Profile pic:
WhatsApp Image 2022-10-27 at 19 09 47

**Comment by Krishna2323** [comment]
Bio:I am a self-taught full-stack developer. I love working on the front-end side and I always try to bring something unique by exploring and playing with advanced CSS styles and effects. I love to tackel tough challenges.
While contributing to Layer5 projects I always learn something valuable.   
Location: Gorakhpur, India
Twitter: https://twitter.com/Web_DevLoving
LinkedIn: https://www.linkedin.com/in/krishnagupta23
Github:https://github.com/Krishna2323
profile-picture

**Comment by Krishna2323** [comment]
@tinniaru3005 ^^^

**Comment by l5io** [comment]
🚀 Preview for commit fa109c574383f5d555dd4e3f4fac7cafb8b97bcb at: https://638b1e8acdadd203544a88ff--layer5.netlify.app

**Comment by eeshaanSA** [comment]
Looks good. @Krishna2323 do you intend the transparent background? An image with a bg would look better.

**Comment by tinniaru3005** [comment]
> Looks good. @Krishna2323 do you intend the transparent background? An image with a bg would look better.

I guess yes as all the other images have a background. Having a background will make it look better.

**Comment by tinniaru3005** [comment]
@Krishna2323 let us know if you want a background or want to keep it transparent.

**Comment by Krishna2323** [comment]
@tinniaru3005, pls add a background if you can because it looks odd.

**Comment by l5io** [comment]
🚀 Preview for commit f4ef565c6708a5990407eb5deb13ab8627f96e0d at: https://638b2af7f783e80b1e8470ca--layer5.netlify.app

**Comment by eeshaanSA** [comment]
LGTM 🚀

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**